_internal/utils/sourceme.R

```diff
@@ -206,9 +61,19 @@ restore_and_run <- function(app_name, username = "insightsengineering", repo = "
   #'
   #' @param app_directory The directory path of the Shiny app.
   #' @param ... Additional arguments passed to `shiny::runApp()`.
-  load_and_run_app <- function(app_directory, ...) {
-    renv::load(app_directory)
-    renv::restore(clean = TRUE, prompt = FALSE)
+  load_and_run_app <- function(app_directory, package_repo, ...) {
+    renv::dependencies()$Package |>
+      tools::package_dependencies() |>
```

Comment:
This is for it to work locally. The CI will not have any problems because it will install the packages from scratch so everything will be updated.

But, if a user already had the `teal.slice` package installed the `renv::dependencies()` will just get the top dependency like `teal` and `teal.modules.general`, and the `teal.slice` will not be updated for the user causing errors sometimes.